packages/app/src/components/views/publication/ArticleView.tsx

```diff
@@ -77,7 +77,7 @@ export const ArticleView: React.FC<ArticleViewProps> = ({ updateChainId }) => {
   }, [setMarkdownArticle])
 
   return (
-    <PublicationPage showCreatePost={false} publication={article?.publication}>
+    <PublicationPage showCreatePost={false} showEditButton={true} publication={article?.publication}>
```

Comment:
@juliopavila I hardcoded this as true for now, but it will need to be dynamic based on whether the user has edit permissions